rackcity/models/log.py

```diff
@@ -17,9 +17,11 @@ class Log(models.Model):
         verbose_name="related user",
         null=True,
     )
-    related_assets = models.ManyToManyField(
+    related_asset = models.ForeignKey(
```

Comment:
per our discussion earlier today, we are only going to link ONE asset to any log (if a change involves multiple assets such as network changes, then multiple logs will be written, one for each affected asset)